src/Prismic/Dom/RichText.php

```diff
@@ -277,7 +277,7 @@ private static function serialize($element, $content, $linkResolver, $htmlSerial
             case 'o-list-item':
                 return nl2br('<li' . $classCode . '>' . $content . '</li>');
             case 'image':
-                $img = '<img src="' . $element->url . '" alt="' . htmlentities($element->alt) . '">';
+                $img = '<img src="' . $element->url . '" alt="' . htmlentities($element->alt ?? '') . '" width="'. $element->dimensions->width .'" height="'. $element->dimensions->height .'">';
```

Comment:
Can we be sure that `dimensions` and the properties `width` and `height` always exists?

If so, then everything is good. Otherwise we could use:

`$element->dimensions?->width ?? ''`

In case this get's changed we should ensure that there is also a testcase for this new scenario

Reply:
The `dimensions.width` and `dimensions.height` properties should always exist in API responses. The `null` fallback is still a good idea just in case someone is working on a very old repository that might be running on an older version of the API.